modules/custom/openy_search/openy_search_api/openy_search_api.module

```diff
@@ -46,3 +47,17 @@ function openy_search_api_openy_search_theme_configuration_alter(&$overrides) {
     'display_search_form' => 1,
   ];
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function openy_search_api_form_views_exposed_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  if ($form['#id'] == 'views-exposed-form-search-content-search-content') {
```

Comment:
Why do we need this altering?

Reply:
This altering to make design similar to YGBW for Open Y Rose theme